examples/psbt.rs

```diff
@@ -0,0 +1,33 @@
+extern crate bitcoin;
+
+use std::{env, process};
+
+use bitcoin::util::psbt::PartiallySignedTransaction;
+use bitcoin::consensus::encode::deserialize;
+
+fn main() {
+
+  let args: Vec<String> = env::args().collect();
+  if args.len() < 2 {
+      eprintln!("not enough arguments. usage: cargo run --example psbt <PSBT>");
+      process::exit(1);
+  }
+  let decoded_psbt = match base64::decode(&args[1]){
```

Comment:
```suggestion
  let decoded_psbt = match base64::decode(&args[1]) {
```
This is why we need a formatter :)

Reply:
damn, clippy results:
```
error: aborting due to previous error; 113 warnings emitted
```
I'll attempt addressing this in another PR.

Reply:
Now we have clippy mfixups erged so it should work for now. But also pls format this specific file with `rustfmt`